gremlin-dotnet/src/Gremlin.Net/Structure/IO/GraphSON/EdgeDeserializer.cs

```diff
@@ -31,26 +31,26 @@ internal class EdgeDeserializer : IGraphSONDeserializer
         public dynamic Objectify(JsonElement graphsonObject, GraphSONReader reader)
         {
             var outVId = reader.ToObject(graphsonObject.GetProperty("outV"));
-            var outVLabel = graphsonObject.TryGetProperty("outVLabel", out var outVLabelProperty)
-                ? outVLabelProperty.GetString()
+            string outVLabel = graphsonObject.TryGetProperty("outVLabel", out var outVLabelProperty)
```

Comment:
(nitpick) The change from `var` to `string` shouldn't be necessary and I'd like to avoid that. With the added `!` behind `GetString()`, my IDE already shows that `outVLabel` is of type `string` if I keep `var` here.
(The same applies to the other places in this diff of course.)

Reply:
Visual Studio 17.9.5 think `GetString()!` is not enough
![image](https://github.com/apache/tinkerpop/assets/7605999/e1350e15-88fd-47f1-ada7-dc8c1b5aa713)
(formatting changed so that everything would fit)

Reply:
> Visual Studio 17.9.5 think GetString()! is not enough

OK, I see. I'm using Rider and that even complains if `string` is used instead of `var`, but then we can stick with `string` of course.